seed/migrations/0210_natural_sort.py

```diff
@@ -0,0 +1,219 @@
+from django.contrib.postgres.operations import CreateCollation
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('seed', '0209_auto_20230929_0959'),
+    ]
+
+    operations = [
+        CreateCollation(
+            'natural_sort',
+            provider='icu',
+            locale='en@colNumeric=yes',
+        ),
```

Comment:
This is where the magic happens

Reply:
wow, never new that was a thing! Go postgres!